addons/storyshots/storyshots-core/src/frameworks/angular/helpers.ts

```diff
@@ -31,11 +31,14 @@ const createComponentFromTemplate = (template: string) => {
   })(componentClass);
 };
 const extractNgModuleMetadata = (importItem: any): NgModule => {
+  const target = importItem && importItem.ngModule ? importItem.ngModule : importItem;
```

Comment:
Instead of checking if `importItem` exists can we instead add an interface for `importItem: any`? I haven't checked but maybe angular already provide such an interface. Or we just write down what we need

```
interface StorybookAngularMetaData {
  ngModule?: AngularModule;
}
```
Though, as I am writing this I just noticed we need dynamic access to `importItem[decoratorKey]` 

@ndelangen haven't we encountered the same type conflict in one of your PRs where we need some fix types and additionally `[key: string]: object` ? 

@pascaliske what do you think?

Reply:
It would definitely be good to type out `importItem` with an interface but we still need to check for the existence to gracefully cover edge cases where the item could be undefined, I think...